src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
-      (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)
-      (present-response/success subscribers)
+      (let [insertions (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)]
```

Comment:
Dropping to java is one of my biggest grumps with Clojure.  A lot of it seems to be touted as a feature, though.

In this case it doesn't mess up the functional flow but we'll have to be careful to stay lispy.